### ISSUE TITLE
Progress?

### ISSUE THREAD

**Issue by rkarlsba** [tomato42/lvmts#1]
Hi

This really looks interesting, but what happened, really? I see the project hasn't been updated for five years…

roy

**Comment by tomato42** [comment]
1. it was more of a proof of concept than anything else
2. the performance was going down the more the volume was fragmented
3. [lvmcache](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Logical_Volume_Manager_Administration/lvm_cache_volume_creation.html) now exists

**Comment by rkarlsba** [comment]
As for 3: lvmcache/dm-cache (or bcache) isn't tiering, it's caching.

**Comment by rkarlsba** [comment]
About performance/fragmentation, if you compare it to like Dell Compellent, it uses an (IIRC) 8MB allocation size for dealing with tiering/dedup, which make the lookuptables smaller.

**Comment by tomato42** [comment]
I tested it with 4MiB chunks and as soon as there were more than few hundred non-continuous chunks on the SSD storage, the read access for data on SSD fell below the performance of HDD. Raising it to even a thousand chunks is still only ~8GiB of data. That's rather pathetic. And shows rather clearly that this is not what the system was designed to do.

if you want to continue the work where I left of, I'd have no problem setting up a github organisation for it and giving you commit access - I see no need for a fork.

Anyway, it was just a hobby project, and now I think my effort is better spent on testing if the file systems don't loose data when HDDs lie about write completion (or power is suddenly cut). Among other things, so that btrfs becomes more trustworthy, so I'm focusing on [fsresck](https://github.com/tomato42/fsresck).

> As for 3: lvmcache/dm-cache (or bcache) isn't tiering, it's caching.

true, but I worked on this with the primary purpose of making often accessed blocks faster, lvmcache does that
